docs/enhancements/label-clusterimagesets.md

```diff
@@ -0,0 +1,96 @@
+---
+title: label-clusterimagesets-with-version-and-arch
+authors:
+  - "@mhrivnak"
+creation-date: 2022-02-23
+last-updated: 2022-02-23
+---
+
+# Label ClusterImageSets with version and arch
+
+
+## Summary
+
+Label each ClusterImageSet resource with two labels: the architecture of the
+release image, and the OpenShift version. This will enable assisted-service to
+understand the releases and find a particular version and architecture when
+needed.
+
+## Motivation
+
+When installing clusters with an architecture other than x86_64,
+assisted-service requires the user to specify OpenShift release images for both
+the x86_64 architecture and the target architecture, at the same version. This
```

Comment:
nit: Use a more generic language in this document (and in the implementation) than "x86_64", e.g. "hub CPU architecture" and "spoke CPU architecture", so this enhancement still make sense when we want to support non-x86_64 hubs